### PR TITLE
vial keychron k8 pro iso/rgb support

### PR DESCRIPTION
**Pull request by cloonix**
Added vial support for Keychron K8 Pro ISO/RGB layout based on adophoxia/vial-qmk/keychron-bluetooth-PR pull request which makes Bluetooth available for new Keychron BT keyboards.

### CODE REVIEW

File: keyboards/keychron/k8_pro/iso/rgb/keymaps/vial/config.h [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+
+//Vial Keyboard UID
+#define VIAL_KEYBOARD_UID {0x5C, 0x1E, 0xC0, 0x71, 0x8A, 0x45, 0x35, 0xAD}
+
+#define VIAL_UNLOCK_COMBO_ROWS { 0, 3 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 13 }
+#define DYNAMIC_KEYMAP_MACRO_COUNT 32
```
Comment (adophoxia) [inline keyboards/keychron/k8_pro/iso/rgb/keymaps/vial/config.h]: Is there a reason this define is added?

File: keyboards/keychron/k8_pro/iso/rgb/config.h [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#ifdef RGB_MATRIX_ENABLE
+/* RGB Matrix driver configuration */
+#    define DRIVER_COUNT 2
+
+#    define DRIVER_ADDR_1 0b1110111
+#    define DRIVER_ADDR_2 0b1110100
+#    define DRIVER_1_LED_COUNT 47
+#    define DRIVER_2_LED_COUNT 41
+#    define RGB_MATRIX_LED_COUNT (DRIVER_1_LED_COUNT + DRIVER_2_LED_COUNT)
```
Comment (adophoxia) [inline keyboards/keychron/k8_pro/iso/rgb/config.h]:
```suggestion
#    define RGB_MATRIX_LED_COUNT 88

**Review by adophoxia — CHANGES_REQUESTED** [review]

File: keyboards/keychron/k8_pro/iso/rgb/keymaps/vial/config.h (hunk 1 above)
Comment (cloonix) [reply keyboards/keychron/k8_pro/iso/rgb/keymaps/vial/config.h]: Updated code and removed personal customization (macro count)

File: keyboards/keychron/k8_pro/iso/rgb/config.h (hunk 2 above)
Comment (cloonix) [reply keyboards/keychron/k8_pro/iso/rgb/config.h]: Took my code from the Keychron/qmk_firmware/bluetooth_playground. Couldn't find a reason for that calculation. Removed it.

**Review by adophoxia — APPROVED** [review]
LGTM!

**Pull request merged**